template/src/ontology/Makefile.jinja2

```diff
@@ -469,6 +469,11 @@ $(COMPONENTSDIR)/%: | $(COMPONENTSDIR)
 	touch $@
 .PRECIOUS: $(COMPONENTSDIR)/%
 
+.PHONY: component-download-mondo.owl
```

Comment:
FYI: The change.

Reply:
This does not make too much sense for ODK - this has to go into the custom makefile of your project!